doc/datavolumes.md

```diff
@@ -38,6 +38,99 @@ The following statuses are possible.
 * Failed: The operation has failed.
 * Unknown: Unknown status.
 
+## Target Storage/PVC
+
+There are two ways to request a storage - by using either the `pvc` or the `storage` section in the DataVolume resource yaml.
+Both result in CDI creating a PVC resource, but there are some differences in how they work.
+
+> [!NOTE] 
+> We recommend using the storage API as it has more convenience features.
+
+### PVC
+The `pvc` type specifies the PersistentVolumeClaim resource that will be created by the CDI. 
+All the parameters of pvc have the semantics of PersistentVolumeClaim parameters, 
+e.g when the volumeMode is not specified the kubernetes default Filesystem is used. The example shows 
+  that a PVC with at least 1Gi of storage and ReadWriteOnce accessMode will be created. 
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: blank-dv-with-pvc
+spec:
+  source:
+    blank: {}
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+```
+
+### Storage
+The `storage` type is similar to `pvc` but it allows you to ommit some parameters.
+
+> [!TIP]
+> With the storage API, CDI computes virtualization-specific defaults for optional
+> fields, otherwise falling back to k8's.
+
+If you skip the `volumeMode` parameter, CDI will search for a default value in the
+StorageProfile. See also: [storage profile documentation](storageprofile.md).
+
+If you skip the `storageClassName` name parameter, CDI will prioritize the default
+virtualization storage class over k8s' default. You can define your default
+virtualization storage class by annotating it with
+`storageclass.kubevirt.io/is-default-virt-class` set to `"true"`.
+
+If you request storage with `volumeMode: fileSystem`, CDI will take the file system
```

Comment:
We should word this slightly differently so say that fs overhead is applied whenever filesystem volumeMode is used in conjunction with the storage api (whether specified explicitly or chosen automatically).  As you currently have it it sounds like this happens only if you include `volumeMode: FileSystem` in the spec.

Reply:
Makes sense :+1: